apis/python/tests/test_experiment_query.py

```diff
@@ -182,6 +182,37 @@ def test_experiment_query_value_filter(soma_experiment):
         assert query.var().concat()["label"].to_pylist() == var_label_values
 
 
+@pytest.mark.parametrize("n_obs,n_vars", [(1001, 99)])
```

Comment:
```suggestion
```

Unused.

Reply:
@nguyenv actually the pre-existing `in` case and my new `not in` case both _fail_ if this parametrization line is removed from their respective test functions

This is because the `soma_experiment` fixture implicitly relies on them as arguments